.taskcluster.yml

```diff
@@ -84,6 +84,7 @@ tasks:
                     --
                     --channel=${browser.channel}
                     --log-wptreport=../artifacts/wpt_report.json
+                    --log-wptscreenshot=../artifacts/wpt_screenshot.db
```

Comment:
I don't really like the `.db` extension, but I guess you've hardcoded this elsewhere now so meh.

Reply:
@jgraham not really. Any suggestions?

Reply:
From IRC:

15:03 < jgraham> Hexcles: Honestly if it's a made-up format I'd be pretty happy with no extension
15:07 < Hexcles> jgraham: hmm what about txt?
15:09 < jgraham> Hexcles: txt is fine […]